src/Microsoft.ML.Transforms/Text/StopWordsRemovingTransformer.cs

```diff
@@ -483,10 +480,25 @@ private bool ResourceExists(StopWordsRemovingEstimator.Language lang)
     }
 
     /// <summary>
-    /// Stopword remover removes language-specific list of stop words (most common words)
-    /// This is usually applied after tokenizing text, so it compares individual tokens
-    /// (case-insensitive comparison) to the stopwords.
+    /// <see cref="IEstimator{TTransformer}"/> for the <see cref="CustomStopWordsRemovingTransformer"/>.
     /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    /// ###  Estimator Characteristics
+    /// |  |  |
+    /// | -- | -- |
+    /// | Does this estimator need to look at the data to train its parameters? | No |
+    /// | Input column data type | Vector of [Text](<xref:Microsoft.ML.Data.TextDataViewType>) |
+    /// | Output column data type | Unknown-sized vector of [Text](<xref:Microsoft.ML.Data.TextDataViewType>) |
+    ///
+    /// The resulting <xref:Microsoft.ML.Transforms.Text.StopWordsRemovingTransformer/> creates a new column, named as specified in the output column name parameters, and
+    /// fills it with vector of words similar to vector of words in input column but removing all, predefined for certain language, texts from it.
+    /// All text comparison made by casting predefined text and text from input column to lower case using casing rules of invariant culture.
+    /// See the See Also section for links to examples of the usage.
```

Comment:
Suggest: The resulting <xref:Microsoft.ML.Transforms.Text.StopWordsRemovingTransformer/> creates a new column, named as specified in the output column name parameter, and fills it with **a** vector of words **containing all of the words** in **the** input column **except the predefined list of stopwords for the specified language.

Reply:
Looks good! Small type: extra ** in there